etc/smashbox.conf.template

```diff
@@ -23,6 +26,7 @@ oc_account_password=""
 # owncloud test server
 # if left blank or "localhost" then the real hostname of the localhost will be set
 oc_server = ''
+oc_root = 'owncloud'
```

Comment:
Thanks!  I wasn't sure how to add oc_root to properties being defined in the conf file.  I'll take a look at making this change.